hc/accounts/tests/test_login.py

```diff
@@ -21,12 +22,21 @@ def test_it_sends_link(self):
 
         ### Assert that a user was created
 
+        user_check = User.objects.filter(email="alice@example.org")
```

Comment:
***CodeQuality: Nitpick***
Since you are getting only one value from the database this would be better as this
`user = User.objects.get(email="alice@example.org")`
So you avoid repitation on line 36